components/style/binding_tools/check_bindings.py

```diff
@@ -0,0 +1,27 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import re
+
+
+def build(objdir, verbose=False):
+    with open(objdir + "/bindings.rs", "r") as bindings:
```

Comment:
You shouldn't be using the objdir here, you should be using local paths. The objdir is pointed to where gecko headers live.